### ISSUE TITLE
cannot ghfetch myself due to my pfp

### ISSUE THREAD

**Issue by olivertzeng** [ghfetch/ghfetch#2]
```
❯ ghfetch olivertzeng
Traceback (most recent call last):
  File "/bin/ghfetch", line 287, in <module>
    main()
  File "/bin/ghfetch", line 284, in main
    print_output(fetched_info)
  File "/bin/ghfetch", line 188, in print_output
    output = image_to_unicode(fetched_info['image'])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/bin/ghfetch", line 156, in image_to_unicode
    r, g, b = char if len(char) == 3 else char[:-1]
                      ^^^^^^^^^
TypeError: object of type 'int' has no len()
```
why???

**Comment by olivertzeng** [comment]
yup it seems that its my pfp's problem
![olivertzeng](https://github.com/ghfetch/ghfetch/assets/86348833/e34a8e0d-108d-42b0-bd80-1d4c8e9cd00c)

**Comment by nullgaro** [comment]
Solved in the new release 1.5.4
